Cargo.toml

```diff
@@ -1,38 +1,88 @@
+# Copyright (C) 2022 Nicolas Stalder AND  Nitrokey GmbH
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
 [package]
 name = "piv-authenticator"
-version = "0.0.0-unreleased"
-authors = ["Nicolas Stalder <n@stalder.io>"]
-edition = "2018"
+version = "0.3.2"
+authors = ["Nicolas Stalder <n@stalder.io>", "Nitrokey GmbH"]
+edition = "2021"
 license = "Apache-2.0 OR MIT"
-repository = "https://github.com/solokeys/piv-authenticator"
+repository = "https://github.com/nitrokey/piv-authenticator"
 documentation = "https://docs.rs/piv-authenticator"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[[example]]
+name = "vpicc"
+required-features = ["vpicc"]
+
+
+[[example]]
+name = "usbip"
+required-features = ["apdu-dispatch", "virt"]
 
 [dependencies]
 apdu-dispatch = { version = "0.1", optional = true }
-delog = "0.1.0"
+delog = { version = "0.1.5", optional = true }
 flexiber = { version = "0.1", features = ["derive", "heapless"] }
 heapless = "0.7"
 hex-literal = "0.3"
-interchange = "0.2.0"
+interchange = "0.2.2"
 iso7816 = "0.1"
-serde = { version = "1", default-features = false }
-trussed = { git = "https://github.com/trussed-dev/trussed" }
-# trussed = { path = "../../../trussed" }
-untrusted = "0.7"
+serde = { version = "1", default-features = false, features = ["derive"] }
+trussed = { version = "0.1", features = ["serde-extensions"] }
+trussed-auth = { version = "0.2" }
+untrusted = "0.9"
+vpicc = { version = "0.1.0", optional = true }
+log = "0.4"
+heapless-bytes = "0.3.0"
+subtle = { version = "2", default-features = false }
+trussed-rsa-alloc = { version = "0.1.0", features = ["raw"] }
```

Comment:
Can we make this algorithm optional, so the entire app has `alloc` optional please.

Reply:
Done